met-web/src/routes/LandingPage/LandingPage.tsx

```diff
@@ -2,16 +2,16 @@ import React from 'react';
 import EnhancedTable from '../../components/layout/Table/Table';
 import Button from '@mui/material/Button';
 import Grid from '@mui/material/Grid';
-import { SearchContainer } from './LandingPageElements';
-import { Container } from '@mui/material';
+import { Container} from '@mui/material';
 import { Link } from 'react-router-dom';
+import TextField from '@mui/material/TextField';
 
 const LandingPage = () => {
     return (
         <Container sx={{ paddingTop: '5em' }}>
             <Grid direction="row" justifyContent="center" alignItems="center" container spacing={2}>
                 <Grid item xs={9}>
-                    <SearchContainer></SearchContainer>
+                    <TextField sx={{ width: '30%', fontSize: '40px' }} placeholder={'Search Engagements by name...'} />
```

Comment:
Let's avoid tapping into width and height props of components. In the case above we can use fullWidth prop and control the width by the xs prop in the Grid item above. This way there's a conflict between the column span of the grid and the textField width